composables/use-host-app-message.ts

```diff
@@ -0,0 +1,37 @@
+import * as v from 'valibot'
+import {
+  type IncomingMessage,
+  type OutgoingMessage,
+  incomingMessageSchema,
+} from '~/types/messages'
+
+export function useHostAppMessage() {
```

Comment:
Είχαμε πει αυτό

```suggestion
export function useHostAppMessaging() {
```

Source: https://app.radicle.at/nodes/seed.radicle.at/rad:z3Makm6fsQQXmpSFE43DZqwupaEhk/tree/src/utils/webview-messaging.ts